openapi/pagopa-arc-be.openapi.yaml

```diff
@@ -683,16 +635,6 @@ components:
           type: boolean
         registeredToMe:
           type: boolean
-    TransactionDetailsDTO:
-      type: object
-      properties:
-        infoTransaction:
-          $ref: "#/components/schemas/InfoTransactionDTO"
-        carts:
-          type: array
-          items:
-            $ref: "#/components/schemas/CartItemDTO"
-          x-field-extra-annotation: "@lombok.Builder.Default"
     InfoTransactionDTO:
```

Comment:
Is InfoTransactionDTO still used? if not, remove it

Reply:
removed